polaris/ocean/convergence/convergence.cfg

```diff
@@ -4,7 +4,7 @@
 convergence_eval_time = 24.0
 
 # Convergence threshold below which a test fails
-convergence_thresh = 1.0
+convergence_thresh = 0.1
```

Comment:
Hopefully, no tests are using this default but I would be curious why it was changed here.